DataManagementSystem/Utilities/CSHelpers.py

```diff
@@ -0,0 +1,235 @@
+"""
```

Comment:
Yes, of course, we should centralize. ConfigurationSystem.Client is the "correct" location, and we should not add new helpers inside the single systems.